brave/common/tor/tor.mojom

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Brave Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+module tor.mojom;
+
+import "mojo/common/file_path.mojom";
+
+const string kTorServiceName = "tor_launcher";
+
+interface TorLauncher {
+  Launch(mojo.common.mojom.FilePath tor_bin, array<string> args)
+    => (bool result);
+
+  // TODO(darkdh): info of callback can be more granular
+  SetCrashHandler() => (int32 pid);
```

Comment:
Can you make this int64, just to be future-proof in case operating systems switch to 64-bit pids?

Reply:
fixed in fea7dca